docs/connected-accounts.rst

```diff
@@ -0,0 +1,15 @@
+Connecting Your Account
+-----------------------
+
+If you are going to import repositories from GitHub, Bitbucket, or GitLab,
+you should connect your Read the Docs account to your repository host first.
+Connecting your account allows for:
+
+* Easier importing of your repositories
+* Automatically configure your repository :doc:`webhooks`
+* Log into Read the Docs with your GitHub, Bitbucket, or GitLab credentials
+
+To connect your account, go to your *Settings* dashboard and select *Connected
+Services*. From here, you'll be able to connect to your GitHub, Bitbucket or GitLab
```

Comment:
I believe you can also just "login" with the accounts and it will connect them. This is probably the most explicit way of documenting doing it though 👍

Reply:
Signing up will connect them. I'll make that a bit more clear.